tests/acceptance/pageObjects/accountPage.js

```diff
@@ -2,7 +2,7 @@ const { join } = require('../helpers/path')
 
 module.exports = {
   url: function () {
-    return join(this.api.launchUrl, '/index.html#/account/')
+    return join(this.api.launchUrl, '/#/account/')
```

Comment:
this change has allowed some tests to pass but the code doesn't seem to have connection with most of the tests that failed previously and now pass.

And also, other tests are still failing with the same issue (login page reappearing).